### ISSUE TITLE
What is the your momentum SGD algorithm used in cnn_train.m and cnn_train_dag.m?

### ISSUE THREAD

**Issue by robot520** [vlfeat/matconvnet#913]
I think it should be:
m(t+1)=A*m(t)-B*Der(w(t)); A>0 and B in [0,1]
W(t+1)=W(t)+m(t+1);
where Der(w(t)) denote the partial derivative of loss function on the mini-batch training data.
However when I read the source code, the implementation may be
m(t+1)=A*m(t)-Der(w(t));
W(t+1)=W(t)+B*m(t+1);
Can you tell me the actual reason?

**Comment by robot520** [comment]
B is called learning rate
A is called momentum paramter

**Comment by lenck** [comment]
It's effectively the same, no? Just a different parametrisation...

**Comment by robot520** [comment]
@lenck Thanks. I may confused by the formulations. In fact, the equation m(t+1)=Am(t)-BDer(w(t)) can be viewed as (1/B)m(t+1) = (A/B)m(t)-Der(w(t)). The parameter (A/B) play the same role as the parameter A in the matconv implementation Therefore, I think your answer is right.
